src/nspawn/nspawn-settings.c

```diff
@@ -673,3 +675,52 @@ int config_parse_oom_score_adjust(
 
         return 0;
 }
+
+int config_parse_cpu_affinity(
```

Comment:
This seems almost the same as `config_parse_exec_cpu_affinity()`. Is it possible to merge this with the function?

Reply:
so, yeah, that's is what I started from actually. But it's nasty since the cpuset stuff needs two fields really instead of one, which means we need to pass it as a structure. Currently that's done through the Settings structure in the nspawn case and the ExecContext structure in the PID 1 case...

i figure we should introduce a new structure for the cpuset stuff that can be embedded in both, but i didn't want to pile on yet another refactor here, and hence I figured better do that as a follow-up.

Note that the OOMScoreAdjust is a similar case btw, though in that case we might be able to simply use a special value outside the normal oom score adjust range as "is set" indication...